src/inference/graph_inference.cpp

```diff
@@ -88,6 +88,14 @@ namespace std {
   };
 }
 
+uint64 HashInt(int x) {
```

Comment:
Furthermore, [some attribution](http://stackoverflow.com/a/12996028/824425) would be nice.